website/source/docs/guides/creating-certificates.html.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: "docs"
+page_title: "Creating Certificates"
+sidebar_current: "docs-guides-creating-certificates"
+description: |-
+  Learn how to create certificates for Consul.
+---
+
+# Creating Certificates
+
+Correctly configuring TLS can be a complex process, especially given the wide
+range of deployment methodologies. This guide will provide you with a
+production ready TLS configuration.
+
+~> Note that while Consul's TLS configuration will be production ready, key
+   management and rotation is a complex subject not covered by this guide.
+   [Vault][vault] is the suggested solution for key generation and management.
+
+The first step to configuring TLS for Consul is generating certificates. In
+order to prevent unauthorized cluster access, Consul requires all certificates
+be signed by the same Certificate Authority (CA). This should be a _private_ CA
+and not a public one like [Let's Encrypt][letsencrypt] as any certificate
+signed by this CA will be allowed to communicate with the cluster.
+
+~> Consul certificates may be signed by intermediate CAs as long as the root CA
+   is the same. Append all intermediate CAs to the `cert_file`.
+
```

Comment:
I still feel that you can follow the [_Guide Template_](https://docs.google.com/document/d/17zk1a_azbAp0ph4_D_xh8E-LJ6tIFR_5C967ik1Rj9c/edit) and insert the following sections:

- Reference Material (you can reference back to /docs/agent/encryption.html)
- Estimated Time to Complete
- Prerequisites (if any)
- Steps